output/cloud/expv2/flush.go

```diff
@@ -19,10 +19,10 @@ type metricsFlusher struct {
 	maxSeriesInSingleBatch     int
 }
 
-// Flush flushes the queued buckets sending them to the remote Cloud service.
-// If the number of time series collected is bigger than maximum batch size than
-// it splits in chunks.
-func (f *metricsFlusher) Flush(ctx context.Context) error {
+// flush flushes the queued buckets sending them to the remote Cloud service.
+// If the number of time series collected is bigger than maximum batch size
+// then it splits in chunks.
+func (f *metricsFlusher) flush(_ context.Context) error {
```

Comment:
Why keep the context if we are not going to use it?

Reply:
Because https://github.com/grafana/k6/pull/3100#discussion_r1228403404 is going to use it

Reply:
It seems to me they will have exactly the same problem ... but let's see I guess :shrug: